templates/home.tmpl

```diff
@@ -1,33 +1,16 @@
-<!DOCTYPE html>
+{{define "title"}}Home{{end}}
 
-<html lang="en">
-    <head>
-        <meta charset="utf-8" />
-        <meta
-            name="description"
-            content="A better way to book club" />
-        <meta
-            name="viewport"
-            content="width=device-width, initial-scale=1" />
+{{define "js"}}<script src="/static/js/home.js"></script>{{end}}
 
-        <title>CerealNotes</title>
+{{define "content"}}
```

Comment:
we should be consistent about tabbing after a `define`. I prefer what you did here where it's tabbed, but in base.tmpl it's not tabbed